CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Added optional body text to `InformationMessage`
 
+### Fixed
+- Fixed incorrect hex codes for button highlights in 5.4.1
```

Comment:
The latest version of components is 5.5.0 so this will need to be 5.5.1

Reply:
This is to say that a bug found in version `5.4.1` has been fixed